app/tests/cognito/test_cognito_client.py

```diff
@@ -0,0 +1,255 @@
+from unittest.mock import call
+from unittest.mock import patch
+
+from cognito.utils.client import Client
+from cognito.utils.client import user_attributes_to_dict
+
+from django.conf import settings
+
+
+def cognito_user(username, managed, for_):
```

Comment:
I think it's good practice if functions contain a verb. Unless you turn this into a fixture of course.

Reply:
Moved to global fixture `cognito_user_response_factory`